adafruit_stmpe610.py

```diff
@@ -1,26 +1,54 @@
-# SPDX-FileCopyrightText: 2017 Jerry Needell for Adafruit Industries
+# SPDX-FileCopyrightText: 2017, 2022 Jerry Needell for Adafruit Industries
 #
 # SPDX-License-Identifier: MIT
-
 """
 `adafruit_stmpe610`
 ====================================================
-
 This is a CircuitPython Driver for the STMPE610 Resistive Touch sensor
 
-* Author(s): Jerry Needell
-"""
+* Author(s): Jerry Needell, CedarGroveMakerStudios
+
+Implementation Notes
+--------------------
+**Hardware:**
+
+**Software and Dependencies:**
 
-# imports
+* Adafruit CircuitPython firmware for the supported boards:
+  https://github.com/adafruit/circuitpython/releases
+"""
 
 import time
 from micropython import const
 
 
-__version__ = "0.0.0-auto.0"
+__version__ = "1.2.7"
```

Comment:
I don't think there is anything that intentionally should change this as part of the actions / CI process. If that did occur automatically I don't think it was meant to. 

I'm pretty sure we need to put the version back to `0.0.0-auto.0`